htdocs/compta/prelevement/class/bonprelevement.class.php

```diff
@@ -1091,6 +1092,7 @@ function Create($banque=0, $agence=0, $mode='real', $format='ALL',$executiondate
 					}
 
 					$this->factures = $factures_prev_id;
+					$this->factures_prev = $factures_prev;
```

Comment:
Because $this->factures_prev is not a property of BonPrelevement, if you need it into the trigger, you must use:

$this->context = array('factures_prev'=>$$factures_prev);
just before the // Call trigger

Adding propeties that are not to be used by triggers as been deprecated when we add the $this->context property that is dedicated for that.

Reply:
Perfect ! I do it as soon as possible